src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -16,12 +16,16 @@ public class ListCommand extends Command {
             + COMMAND_WORD + "\n\n"
             + "Example: " + COMMAND_WORD;
     public static final String MESSAGE_SUCCESS = "Listed all persons";
-
+    public static final String MESSAGE_EMPTY_LIST = "No clients found, start adding clients using `add`!";
```

Comment:
I think using the backticks won't show it as code in the actual output box since it's not formatted against markdown. Perhaps something like this would be better.
```suggestion
    public static final String MESSAGE_EMPTY_LIST = "No clients found, start adding clients using the Add command!";
```

Reply:
Yup I think this makes more sense, thanks.